### PR TITLE
Added functionality to capital works page

### PR DESCRIPTION
**Pull request by stuartmccoll**
This pull request adds functionality to the project capital works page.

Note that, like the other file upload pages, I haven't tested the non-JS journey. I think we should pick this up separately.

### CODE REVIEW

File: app/controllers/project/project_capital_works_controller.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  include ProjectContext
+
+  def update
+
+    logger.debug "Updating capital works for project ID: #{@project.id}"
+
+    @project.validate_capital_work = true
+
+    if params[:project].present?
+      @project.validate_capital_work_file = true if params[:project][:capital_work] == "true"
```
Comment (gidsg) [inline app/controllers/project/project_capital_works_controller.rb]: Quite possibly missing something here, but it looks like we're setting this value to `true` twice? Can one of these be deleted or am I missing something?

File: app/views/project/project_capital_works/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+                  <th scope="col" class="govuk-table__header">Condition Survey File</th>
+
+                </tr>
+              </thead>
+              <tbody class="govuk-table__body">
+              <tr>
+                <td>
+                  <%# TODO: Link to download path (currently not working due to PaaS permissions %>
+                  <a href="#">
+                    <%= @project.capital_work_file.blob.filename %>
```
Comment (gidsg) [inline app/views/project/project_capital_works/show.html.erb]: This link should work now, as we have resolved the TODO. You can use a link something like: `<%=link_to(@project.capital_work_file.blob.filename, rails_blob_path(project.capital_work_file, disposition: "attachment")) %>`

File: app/controllers/project/project_capital_works_controller.rb (hunk 1 above)
Comment (stuartmccoll) [reply app/controllers/project/project_capital_works_controller.rb]: This line is to validate that the capital work file is present, which we only want to do if the `capital_work` value passed in is true, i.e. a user has selected the 'Yes, my project involves capital work' option.

Line 8 is to validate that the value passed in the `capital_work` param is either `true` or `false`, as per the column on the database.

File: app/views/project/project_capital_works/show.html.erb (hunk 2 above)
Comment (stuartmccoll) [reply app/views/project/project_capital_works/show.html.erb]: Done 👍.

File: app/views/project/project_capital_works/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+        </div>
+
+        <div class="govuk-radios__item">
+
+          <%= f.radio_button :capital_work,
+                             "true",
+                             class: "govuk-radios__input",
+                             "data-aria-controls" => "conditional-project_capital_work_true"
+          %>
+          <%= f.label :capital_work, "Yes, capital work is part of my project", class: "govuk-label govuk-radios__label" %>
```
Comment (gidsg) [inline app/views/project/project_capital_works/show.html.erb]: this label does not match the associated button, I think it should be
```suggestion
          <%= f.label :capital_work_true, "Yes, capital work is part of my project", class: "govuk-label govuk-radios__label" %>
```

File: app/views/project/project_capital_works/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+        By capital work we mean repair and conservation works, new building work,
+        digitisation of objects, or the conservation of artefacts and objects.
+      </span>
+
+      <div class="govuk-radios govuk-radios--conditional" data-module="govuk-radios">
+
+        <div class="govuk-radios__item">
+
+          <%= f.radio_button :capital_work, "false", class: "govuk-radios__input" %>
+          <%= f.label :capital_work, "No, capital work is not part of my project", class: "govuk-label govuk-radios__label" %>
```
Comment (gidsg) [inline app/views/project/project_capital_works/show.html.erb]: I think label does not match the radio button here.
```suggestion
          <%= f.label :capital_work_false, "No, capital work is not part of my project", class: "govuk-label govuk-radios__label" %>
```

**Comment by gidsg** [comment]
When I try to upload a file on this branch, when it gets to the end of the upload the page hangs. The way we have done this on the other page is to do a normal server side redirect in the controller which rails does after the XHR request but doesn't seem to be happening in this case (at least for me)
Screenshot 2020-01-13 at 09 33 17

**Comment by gidsg** [comment]
> When I try to upload a file on this branch, when it gets to the end of the upload the page hangs. The way we have done this on the other page is to do a normal server side redirect in the controller which rails does after the XHR request but doesn't seem to be happening in this case (at least for me)
> Screenshot 2020-01-13 at 09 33 17

OK I think this is a broder issue I have raised as #124

**Pull request merged**